addons/website/static/src/services/website_service.js

```diff
@@ -35,7 +35,7 @@ export const websiteService = {
         let lastUrl;
         let websiteRootInstance;
         let Wysiwyg;
-        let isRestrictedEditor;
+        let isRestrictedEditor;websiteService
```

Comment:
:thinking: Don't forget to review your PR yourself before marking it ready :wink: